### ISSUE TITLE
Postman can't complete the oauth2 login token by connecting Sample Authorization Server

### ISSUE THREAD

**Issue by ZhuBicen** [spring-projects-experimental/spring-authorization-server#131]
**Describe the bug**

Postman can not complete Oauth2 login, by connecting the Sample authoization server
Exception occured: ` java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonIncludeProperties`

**To Reproduce**
0. Using latest commit: Polish gh-117 `628c8bece35ea4115cf1a0a5ffcc021bb8234f38` 
1. add `127.0.0.1 auth-server` to `/etc/hosts/`
2. run ` gradle -b samples/boot/oauth2-integration/authorizationserver/spring-security-samples-boot-oauth2-integrated-authorizationserver.gradle bootRun`
2. Fill the Postman oauth2 login as below:
![image](https://user-images.githubusercontent.com/1465566/95754911-4ed44180-0cd6-11eb-9aa4-303b0b9b0e51.png)
3. Click "Request Token"
4. Input the username/password: `user1/password`
5. But it failed as below
![image](https://user-images.githubusercontent.com/1465566/95754956-5b589a00-0cd6-11eb-8c0f-7e2cde14c042.png)
6. The console:
![image](https://user-images.githubusercontent.com/1465566/95755873-87284f80-0cd7-11eb-8868-482bebb3c335.png)

**Expected behavior**
Postman should get the expected access token

**Sample**

Just try the provided sample server: `samples/boot/oauth2-integration/authorizationserver`
```
2020-10-13 23:19:27.785 ERROR 10356 --- [nio-9000-exec-7] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Filter execution threw an exception] with root cause

java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonIncludeProperties
        at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.findPropertyInclusionByName(JacksonAnnotationIntrospector.java:321) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.cfg.MapperConfigBase.getDefaultPropertyInclusions(MapperConfigBase.java:685) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BasicSerializerFactory.buildMapSerializer(BasicSerializerFactory.java:827) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BasicSerializerFactory.buildContainerSerializer(BasicSerializerFactory.java:617) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BeanSerializerFactory._createSerializer2(BeanSerializerFactory.java:200) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BeanSerializerFactory.createSerializer(BeanSerializerFactory.java:169) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider._createUntypedSerializer(SerializerProvider.java:1474) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider._createAndCacheUntypedSerializer(SerializerProvider.java:1442) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider.findValueSerializer(SerializerProvider.java:562) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider.findTypedValueSerializer(SerializerProvider.java:844) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ObjectWriter$Prefetch.forRootType(ObjectWriter.java:1477) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ObjectWriter.forType(ObjectWriter.java:422) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at org.springframework.http.converter.json.AbstractJackson2HttpMessageConverter.writeInternal(AbstractJackson2HttpMessageConverter.java:336) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.http.converter.AbstractGenericHttpMessageConverter.write(AbstractGenericHttpMessageConverter.java:104) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.security.oauth2.core.http.converter.OAuth2AccessTokenResponseHttpMessageConverter.writeInternal(OAuth2AccessTokenResponseHttpMessageConverter.java:99) ~[spring-security-oauth2-core-5.4.0.jar:5.4.0]
        at org.springframework.security.oauth2.core.http.converter.OAuth2AccessTokenResponseHttpMessageConverter.writeInternal(OAuth2AccessTokenResponseHttpMessageConverter.java:48) ~[spring-security-oauth2-core-5.4.0.jar:5.4.0]
        at org.springframework.http.converter.AbstractHttpMessageConverter.write(AbstractHttpMessageConverter.java:227) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.security.oauth2.server.authorization.web.OAuth2TokenEndpointFilter.sendAccessTokenResponse(OAuth2TokenEndpointFilter.java:183) ~[spring-security-oauth2-authorization-server-0.0.2-SNAPSHOT.jar:0.0.2-SNAPSHOT]
        at org.springframework.security.oauth2.server.authorization.web.OAuth2TokenEndpointFilter.doFilterInternal(OAuth2TokenEndpointFilter.java:162) ~[spring-security-oauth2-authorization-server-0.0.2-SNAPSHOT.jar:0.0.2-SNAPSHOT]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
....
....
Caused by: java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonIncludeProperties
        at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.findPropertyInclusionByName(JacksonAnnotationIntrospector.java:321) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.findPropertyInclusionByName(AnnotationIntrospectorPair.java:133) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.cfg.MapperConfigBase.getDefaultPropertyInclusions(MapperConfigBase.java:685) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BasicSerializerFactory.buildMapSerializer(BasicSerializerFactory.java:827) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BasicSerializerFactory.buildContainerSerializer(BasicSerializerFactory.java:617) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BeanSerializerFactory._createSerializer2(BeanSerializerFactory.java:200) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.BeanSerializerFactory.createSerializer(BeanSerializerFactory.java:169) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider._createUntypedSerializer(SerializerProvider.java:1474) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider._createAndCacheUntypedSerializer(SerializerProvider.java:1422) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.SerializerProvider._findExplicitUntypedSerializer(SerializerProvider.java:1391) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.hasSerializerFor(DefaultSerializerProvider.java:260) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at com.fasterxml.jackson.databind.ObjectMapper.canSerialize(ObjectMapper.java:3325) ~[jackson-databind-2.12.0-rc1.jar:2.12.0-rc1]
        at org.springframework.http.converter.json.AbstractJackson2HttpMessageConverter.canWrite(AbstractJackson2HttpMessageConverter.java:199) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.http.converter.AbstractGenericHttpMessageConverter.canWrite(AbstractGenericHttpMessageConverter.java:76) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.getProducibleMediaTypes(AbstractMessageConverterMethodProcessor.java:368) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:217) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor.handleReturnValue(HttpEntityMethodProcessor.java:219) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:82) ~[spring-web-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:123) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:888) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:802) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1061) ~[spring-webmvc-5.3.0-RC1.jar:5.3.0-RC1]
        ... 82 common frames omitted
```

**Comment by zyjibmcn** [comment]
I hit the same issue, it was caused by multiple versions of Jackson dependency in the classpath. After I specified a specific version of Jackson, the issue resolved.

**Comment by jgrandja** [comment]
Thanks for the report @ZhuBicen. As @zyjibmcn mentioned, this has been resolved via 1428760. Pull the latest and it will work.
